web/src/pages/orders/new/hooks.js

```diff
@@ -0,0 +1,58 @@
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useRouter } from 'next/router'
+import { useForm } from 'react-hook-form'
+import * as yup from 'yup'
+
+import { orderApi } from '@/hooks/api/orderApi'
+import { useCategories } from '@/hooks/redux/useCategories'
+import { useHandleError } from '@/hooks/useHandleError'
+
+const schema = yup.object({
+    store_id: yup.number().oneOf([1, 2, 3]).required(),
+    category_label: yup.string().required(),
+    job_order: yup.number(),
+    brand: yup.string().required(),
+    part_models: yup.string(),
+    model: yup.string(),
+    downpayment: yup.number().default(0),
+    quantity: yup.number().required(),
+    status: yup
+        .string()
+        .oneOf(['open', 'in-transit', 'delivered', 'cancelled', 'onhold'])
+        .default('open'),
+    link: yup.string(),
+    notes: yup.string(),
+    image_paths: yup.array().of(yup.string()),
+    order_at: yup.date().default(() => new Date()),
+})
+
+export function useHooks() {
+    const router = useRouter()
+    const { handleError } = useHandleError()
+    const [createOrderMutation] = orderApi.useCreateOrderMutation()
+    const { categories } = useCategories()
+    const {
+        register,
+        formState: { errors },
+        handleSubmit,
+    } = useForm({ resolver: yupResolver(schema) })
+    const onSubmit = async (data) => {
+        data.order_at = new Date(data.order_at).toISOString().slice(0, 19).replace("T", " ");
```

Comment:
instead of manually formatting, can you try like this.
```suggestion
<DatePicker 
  label="Date" 
  name="order_at" 
  dateFormat="yyyy-MM-dd"
  {...formState} 
/>
```